packages/ses/package.json

```diff
@@ -55,24 +55,34 @@
         "default": "./dist/ses.cjs"
       }
     },
+    "./hermes": {
+      "require": {
+        "types": "./dist/types.d.cts",
+        "default": "./dist/ses-hermes.cjs"
+      }
+    },
     "./tools.js": "./tools.js",
     "./assert-shim.js": "./assert-shim.js",
     "./lockdown-shim.js": "./lockdown-shim.js",
     "./compartment-shim.js": "./compartment-shim.js",
     "./package.json": "./package.json"
   },
   "scripts": {
-    "build": "node scripts/bundle.js",
+    "build:vanilla": "node scripts/bundle.js",
+    "build:hermes": "SES_BUILD_TYPE=hermes node scripts/bundle.js",
+    "build": "yarn build:vanilla && yarn build:hermes",
     "clean": "rm -rf dist",
     "cover": "c8 ava",
     "demo": "python3 -m http.server",
     "lint": "yarn lint:types && yarn lint:eslint",
     "lint-fix": "eslint --fix .",
     "lint:eslint": "eslint .",
     "lint:types": "tsc",
-    "prepare": "npm run clean && npm run build",
+    "prepare": "yarn clean && yarn build",
     "qt": "ava",
     "test": "tsd && ava",
+    "test:hermes": "./scripts/hermes.sh",
+    "test:create-hermes-bin-symlinks": "./scripts/hermes-bin-symlinks.sh",
```

Comment:
also not particularly windows-friendly 😄

Reply:
nb: `test:create-hermes-bin-symlinks` [removed](https://github.com/endojs/endo/pull/2334#discussion_r1765232235)

Reply:
i see your point

![image](https://github.com/user-attachments/assets/fd46666b-e5e3-451f-a6b1-d08150aecfeb)

and others [set up react native on windows](https://reactnative.dev/docs/set-up-your-environment?os=windows) too

i'll create follow-up issue, did you have a solution in mind